Android/app/src/main/java/fashiome/android/activities/ProductFormActivity.java

```diff
@@ -380,7 +380,7 @@ public void onConnected(Bundle dataBundle) {
         if (location != null) {
             Toast.makeText(this, "GPS location was found! Lat:"+ location.getLatitude() +" Long:"+location.getLongitude(), Toast.LENGTH_SHORT).show();
             LatLng latLng = new LatLng(location.getLatitude(), location.getLongitude());
-            product.setAddress(new Address(latLng));
+            product.setAddress(new Address(Item.getRandomLocation(latLng.latitude, latLng.longitude,5000)));
```

Comment:
I wanted to add new products with pics but as they all get loaded on my current loc I am unable to view all of them

You can remove the random method now

Thanks,
Rakhee

On Mar 19, 2016, at 2:55 PM, Deepak Barge <notifications@github.com<mailto:notifications@github.com>> wrote:


In Android/app/src/main/java/fashiome/android/activities/ProductFormActivity.java<https://github.com/deborshisaha/ModaRenta/pull/57#discussion_r56755691>:

> @@ -380,7 +380,7 @@ public void onConnected(Bundle dataBundle) {
>          if (location != null) {
>              Toast.makeText(this, "GPS location was found! Lat:"+ location.getLatitude() +" Long:"+location.getLongitude(), Toast.LENGTH_SHORT).show();
>              LatLng latLng = new LatLng(location.getLatitude(), location.getLongitude());
> -            product.setAddress(new Address(latLng));
> +            product.setAddress(new Address(Item.getRandomLocation(latLng.latitude, latLng.longitude,5000)));


Is this adding a random location still ? Product form activity is working fine with users current location why we need random location ?

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/deborshisaha/ModaRenta/pull/57/files/b5a0e76ed0e25c34f4b9737bf76df95ed8676ffb#r56755691>